transform/mattermost-analytics/models/intermediate/product/licenses/_licenses_servers/int_cloud_servers.sql

```diff
@@ -0,0 +1,12 @@
+with mm_telemetry_prod_license as (
+    select timestamp::date as installation_date
+        , installation_id as installation_id
+        , server_id as server_id
+        , license_name as license_name
+    from {{ ref('stg_mm_telemetry_prod__license')}} 
+    where license_name in ('CLD')
```

Comment:
Is this documented somewhere?

Reply:
Updated and removed this filter, following a conversation with Nick.